src/visit_mut.rs

```diff
@@ -1,25 +1,108 @@
+use proc_macro2::Ident;
 use quote::quote;
 use syn::punctuated::Punctuated;
 use syn::visit_mut::VisitMut;
 use syn::{
-    Arm, Expr, ExprAssign, ExprBinary, ExprCall, ExprClosure, ExprField, ExprForLoop, ExprIf,
-    ExprLet, ExprLoop, ExprMatch, ExprMethodCall, ExprPath, ExprUnary, ExprWhile, Field, Fields,
-    FnArg, ItemConst, ItemEnum, ItemFn, ItemImpl, ItemStatic, ItemStruct, ItemTrait, ItemType,
-    ItemUnion, Macro, Member, Pat, PatIdent, PatTuple, PatType, Path, PathSegment, ReturnType,
-    Signature, Token, Type, Variant,
+    Arm, ConstParam, Expr, ExprAssign, ExprBinary, ExprCall, ExprClosure, ExprField, ExprForLoop,
+    ExprIf, ExprLet, ExprLoop, ExprMatch, ExprMethodCall, ExprPath, ExprUnary, ExprWhile, Field,
+    Fields, FnArg, ImplItem, Item, ItemConst, ItemEnum, ItemFn, ItemImpl, ItemStatic, ItemStruct,
+    ItemTrait, ItemType, ItemUnion, Macro, Member, Pat, PatIdent, PatTuple, PatType, Path,
+    PathSegment, ReturnType, Signature, Token, Type, TypeParam, Variant,
 };
 
 use crate::ident_visitor::IdentVisitor;
 
+fn get_ident(item: &Item) -> Option<syn::Ident> {
```

Comment:
`get_ident` and `get_impl_item_ident` are just utils for sorting, right? Maybe this can be made clear, e.g. a doc comment or moving to a different file. Or how about naming them `get_ident_sort_key` etc.?

Reply:
I personally think that they are not limited to sorting and can be used for others things in the future and I think their names explain exactly their role.